egs/csj/ASR/prepare.sh

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env bash
+# We assume the following directories are downloaded.
+#
+#  - $csj_dir
+#     CSJ is assumed to be the USB-type directory, which should contain the following subdirectories:- 
+#     - DATA (not used in this script)
+#     - DOC (not used in this script)
+#     - MODEL (not used in this script)
+#     - MORPH
+#       - LDB (not used in this script)
+#       - SUWDIC (not used in this script)
+#       - SDB
+#         - core
+#           - ...
+#         - noncore
+#           - ...
+#     - PLABEL (not used in this script)
+#     - SUMMARY (not used in this script)
+#     - TOOL (not used in this script)
+#     - WAV
+#       - core
+#         - ...
+#       - noncore
+#         - ...
+#     - XML (not used in this script)
+#
+#  - $musan_dir
+#      This directory contains the following directories downloaded from
+#       http://www.openslr.org/17/
+#     - music
+#     - noise
+#     - speech
+# 
+# By default, this script produces the original transcript like kaldi and espnet. Optionally, you
+# can generate other transcript formats by supplying your own config files. A few examples of these
+# config files can be found in local/conf.
+
+set -eou pipefail
+
+nj=8
+stage=-1
+stop_stage=100
+
+csj_dir=/mnt/minami_data_server/t2131178/corpus/CSJ
+musan_dir=/mnt/minami_data_server/t2131178/corpus/musan/musan
+trans_dir=$csj_dir/retranscript
+csj_fbank_dir=/mnt/host/csj_data/fbank
+musan_fbank_dir=$musan_dir/fbank
+csj_manifest_dir=data/manifests
+musan_manifest_dir=$musan_dir/manifests
+
+. shared/parse_options.sh || exit 1
+
+mkdir -p data
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%d %H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+if [ $stage -le 1 ] && [ $stop_stage -ge 1 ]; then 
+    log "Stage 1: Prepare CSJ manifest"
+    # If you want to generate more transcript modes, append the path to those config files at c.
+    # Example: lhotse prepare csj $csj_dir $trans_dir $csj_manifest_dir -c local/conf/disfluent.ini
+    # NOTE: In case multiple config files are supplied, the second config file and onwards will inherit
+    #       the segment boundaries of the first config file. 
+    lhotse prepare csj $csj_dir $trans_dir $csj_manifest_dir -j 4
```

Comment:
Shall we place a `.done` file inside `$csj_manifest_dir/`, like line 73 in this file?